src/scripts/main.js

```diff
@@ -1,7 +1,30 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const notification = document.createElement('div');
+
+  notification.classList.add('notification', type);
+
+  const titleElement = document.createElement('h2');
+
+  titleElement.classList.add('title');
+  titleElement.innerText = title;
+
+  const descriptionElement = document.createElement('p');
+
+  descriptionElement.innerText = description;
+
+  notification.appendChild(titleElement);
+  notification.appendChild(descriptionElement);
+
+  notification.style.top = `${[posTop]}px`;
+  notification.style.right = `${[posRight]}px`;
```

Comment:
There is a minor issue with the use of array brackets around `posTop` and `posRight` in the style assignment. These should be removed to correctly set the `top` and `right` styles as numbers, not arrays. Change `${[posTop]}px` to `${posTop}px` and `${[posRight]}px` to `${posRight}px`.